style.css

```diff
@@ -3,28 +3,41 @@ body
   margin: 0;
   padding: 0;
   background: #FFFFFF;
-  position: relative;
+  font-family: 'Roboto', sans-serif;
+  font-size: 13px;
+  width: 100%;
+}
+
+.logoPicture
+{
+	width: 9%;
+	padding-right: 20px;
 }
 
 nav
 {
 	border-bottom: 1px solid #d9dadb;
+	justify-content: space-around;
 }
 
 nav a
 {
-	color: #686868;
+	color: #636262;
+	width: 10%;
+	padding-right: 40px;
```

Comment:
Would it be possible to bundle these two padding values in a single line? If so, how?